tests/terraform/checks/resource/azure/example_AzureSqlPoolEncryption/main.tf

```diff
@@ -0,0 +1,22 @@
+## SHOULD PASS: Explicit false
+resource "synapse_sql_pool" "pass" {
+  name                = "sqlPool1"
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = azurerm_resource_group.rg.location
+  data_encrypted       = true
+}
+
+## SHOULD PASS: Default false
+resource "synapse_sql_pool" "fail1" {
```

Comment:
if it should pass, maybe call it `pass2` instead of `fail1`?

Reply:
It should fail, wrong comment